examples/MultiXcodeProject Example/Examples.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this file be ignored?

Reply:
Don't think so, we probably want to allow developers to run the examples without dirtying the repo.